visualization/app/codeCharta/util/customConfigBuilder.ts

```diff
@@ -72,6 +72,7 @@ function initializeAppSettings(target: CustomConfig) {
 		layoutAlgorithm: undefined,
 		maxTreeMapFiles: 0,
 		sharpnessMode: undefined,
```

Comment:
Please make sure, that old custom configs without the new property can still be loaded and importet.

Reply:
I will test it!

Reply:
It works :)